host/bcm283x/gpio_test.go

```diff
@@ -130,6 +130,13 @@ func TestPin(t *testing.T) {
 	if d := p.Read(); d != gpio.Low {
 		t.Fatal(d)
 	}
+	// gpio.PinIn.Pull()
+	if d := p.Pull(); d != gpio.Float {
```

Comment:
Did you mean PullNoChange?

Reply:
Nope, in bcm2711, we can (finally) read the state of pull resistors. Since the pin was previously set to Float, it should give us the correct result on a bcm2711.

Reply:
Ah because this code runs in RPi4 mode, right. Thanks!